doc/user/flycheck-versus-flymake.rst

```diff
@@ -44,71 +37,68 @@ Please do **not** use this table alone to make your personal judgment.  Read the
-|Error message display  |Tooltip, echo area,    |Tooltip only [#]_      |
-|                       |fully customizable     |                       |
-+-----------------------+-----------------------+-----------------------+
-|List of all errors     |yes                    |**no**                 |
-+-----------------------+-----------------------+-----------------------+
-|Resource consumption   |low                    |**high**               |
-+-----------------------+-----------------------+-----------------------+
-|Unit tests             |all syntax checkers,   |**none?**              |
-|                       |large parts of         |                       |
-|                       |internals              |                       |
-+-----------------------+-----------------------+-----------------------+
++---------------------------+-----------------------+-----------------------+
+|                           |Flycheck               |Flymake                |
++===========================+=======================+=======================+
+|Supports Emacs versions    ||min-emacs|            |22+                    |
++---------------------------+-----------------------+-----------------------+
+|Built-in                   |no [#]_                |yes                    |
++---------------------------+-----------------------+-----------------------+
+|`Enables automatically if  |yes                    |no                     |
+|possible <Enabling Syntax  |                       |                       |
+|Checking_>`_               |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|Checks after               |save, newline, change  |newline, change        |
++---------------------------+-----------------------+-----------------------+
+|Checks in background       |yes                    |yes                    |
++---------------------------+-----------------------+-----------------------+
+|`Automatic syntax checker  |by major mode and      |user-defined [#]_      |
+|selection <Syntax checker  |custom predicates      |                       |
+|selection_>`_              |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|`Manual syntax checker     |yes                    |yes                    |
+|selection <Manual          |                       |                       |
+|Selection_>`_              |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|`Multiple syntax checkers  |yes                    |yes                    |
+|per buffer`_               |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|`Supported languages`_     |>50                    |2 [#]_                 |
```

Comment:
I think wrong, but I don't know that exact number right now

Reply:
So at least 4, plus whatever `flymake-easy` supported.